program/Cargo.toml

```diff
@@ -2,26 +2,28 @@
 name = "spl-token-metadata-example"
 version = "0.3.0"
 description = "Solana Program Library Token Metadata Example Program"
-authors = ["Solana Labs Maintainers <maintainers@solanalabs.com>"]
-repository = "https://github.com/solana-labs/solana-program-library"
-license = "Apache-2.0"
-edition = "2021"
+documentation = "https://docs.rs/spl-token-group-interface"
+authors = { workspace = true }
+repository = { workspace = true }
+homepage = { workspace = true }
+license = { workspace = true }
+edition = { workspace = true }
 
 [features]
 no-entrypoint = []
 test-sbf = []
 
 [dependencies]
-solana-program = "2.1.0"
-spl-token-2022 = { version = "6.0.0", path = "../../token/program-2022", features = ["no-entrypoint"] }
+solana-program = "2.2.1"
+spl-token-2022 = { version = "6.0.0", features = ["no-entrypoint"] }
 spl-token-metadata-interface = { version = "0.6.0", path = "../interface" }
-spl-type-length-value = { version = "0.7.0", path = "../../libraries/type-length-value" }
-spl-pod = { version = "0.5.0", path = "../../libraries/pod" }
+spl-type-length-value = "0.7.0"
+spl-pod = "0.5.0"
 
 [dev-dependencies]
-solana-program-test = "2.1.0"
-solana-sdk = "2.1.0"
-spl-token-client = { version = "0.13.0", path = "../../token/client" }
+solana-program-test = "2.2.0"
+solana-sdk = "2.2.1"
+spl-token-client = "0.13.0"
 test-case = "3.3"
 
 [lib]
```

Comment:
I can't comment on the line below, but on the program we could leave `"cdylib"` only.

Reply:
Done with 88b2596